lib/cqm/converter/hds_record.rb

```diff
@@ -77,28 +78,17 @@ def to_qdm(record)
         patient.data_elements << QDM::PatientCharacteristicExpired.new(expired_datetime: expired_datetime)
       end
 
-      # Convert patient characteristic payer.
```

Comment:
Does this bit no longer need to be converted?

Reply:
No, I didn't think it was one to one enough, so I moved it to extended data.